m2cgen/interpreters/c/code_generator.py

```diff
@@ -45,6 +52,21 @@ def add_var_declaration(self, expr):
         self.add_code_line(tpl(var_name=var_name, size=size))
         return var_name
 
+    def add_var_assignment(self, var_name, value, expr):
```

Comment:
The method signature has become obscure. It's no longer clear what arguments are for. We have `var_name` and `value`, so what is `expr` and what's its role in variable assignment? It's just a bit misleading.

Reply:
Done :) thanks!